source/index.html.md

```diff
@@ -1301,6 +1343,79 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
     <div class="type">May not be all digits, all same character, or all sequential characters (e.g. "abc")</div>
     <div class="type">Optional</div>
   </dd>
+  <dt>travel
+    <span class="type">[\x20-\x7E]{2,22}
+      <a target="_blank" href="https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">
+        ASCII printable characters
+      </a>
+    </span>
+  </dt>
+  <dd>
+    Text about travel will go here
+  </dd>
+</dl>
+
+##### Travel addendum data
+
+For service type Lodging the following fields are relevant
+
+<dl class="dl-vertical">
+  <dt>
+    travel[lodging][no_show]
+    <span class="type">(true|false)</span>
+  </dt>
+  <dd>
+    An indicator that the cardholder is being billed for a reserved room that was not actually
+    rented (AKA a "no-show" charge). The issuer may provide this information on the customer
+    statement or use it to enhance customer service information.
+  </dd>
+  <dt>
+    travel[lodging][extra_charge]
+    <span class="type">(<code>no_extra_charge</code>|<code>restaurant</code>|<code>gift_shop</code>|<code>mini_bar</code>|<code>telephone</code>|<code>other</code>|<code>laundry</code>)</span>
+  </dt>
+  <dd>
+    This field may be used to indicate the type of additional charges added to a customer's bill
+    after check-out. Each position in the field can be used to indicate a type of charge. If there
+    are less than six additional charges, they must be left-justified, space-filled or zero-filled
+    to the right.
+  </dd>
```

Comment:
`s/extra_charge/extra_charges/`?

Could we accept `travel[lodging][extra_charge][1]`, `travel[lodging][extra_charge][2]`, ..., `travel[lodging][extra_charge][6]`?

Reply:
Or maybe nest the dict one lay further?
```
travel[lodging][extra_charges][restaurant]=true
travel[lodging][extra_charges][gift_shop]=false # Optionally either false or left out
travel[lodging][extra_charges][other]=true
travel[lodging][extra_charges][laundry]=true
```

If "extra charges" is not some generally agreed upon terminology, then perhaps the shorter `travel[lodging][extras][restaurant]`?